resources/assets/scripts/app/directives/colorbox.coffee

```diff
@@ -0,0 +1,26 @@
+# Pony.fm - A community for pony fan music.
+# Copyright (C) 2015 Zeusking19
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Based on http://stackoverflow.com/questions/14641791/how-to-use-colorbox-with-angular-js
+
+angular.module('ponyfm').directive 'colorbox', ->
+  {
+    restrict: 'AC'
+    link: (scope, element, attrs) ->
+      $(element).colorbox({maxWidth:'90%', maxHeight:'90%'})
+      return
```

Comment:
This line is unnecessary - there's no value being returned or flow control here, and CoffeeScript turns the last line of every function into a return statement anyway.